composer/package-lock.json

```diff
@@ -11408,8 +11408,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
```

Comment:
Shall we revert this change done to the package-lock.json file? I think you have mistakenly committed it too

Reply:
Yeas. I have reverted it. Committed mistakenly.